odoo/addons/base/models/res_country.py

```diff
@@ -90,7 +90,7 @@ def _name_search(self, name, domain=None, operator='ilike', limit=None, order=No
         if len(name) == 2:
             ids = list(self._search([('code', 'ilike', name)] + domain, limit=limit, order=order))
 
-        search_domain = [('name', operator, name)]
+        search_domain = [('name', operator, name.capitalize())]
```

Comment:
Techincally, you shouldn't `capitalize` for operator `'like'`
maybe only `capitalize` for `=` and `!=`?